<!-- Launch file to move on a preprogrammed trajectory -->

<launch>
	<group ns="ias">
	
	
  		<arg name="arm_typ" default="PSM1"/>
		<arg name="arm_name" default="arm_1"/>
	
			<include file="$(find irob_autosurg)/config/dvrk_topic_names.xml" />  	
  	
			<node name="dvrk_interface" pkg="irob_autosurg" 
			type="dvrk_interface" output="screen">
  		
  				<param name="arm_typ" type="string" value="$(arg arm_typ)" />
				<param name="arm_name" type="string" value="$(arg arm_name)" />
				
				
	</node>
  	</group>
</launch>
